css/styles.css

```diff
@@ -382,3 +382,161 @@ section.padded {
   margin-bottom: 8px;
   color: #555;
 }
+
+.categories {
+  display: block;
+  position: relative;
+  text-align: center;
+  font-weight: bolder;
+  font-size: 14px;
+}
+
+.tab-1 {
+  margin-right: 30px;
+}
+
+.tab-2 {
+  margin-left: 30px;
+}
+
+.categories a:hover {
+  color: #4c884c;
+}
+
+.categories a:active {
+  color: #4c884c;
+}
+
+.tab-container1 {
+  display: block;
+  padding-left: 16px;
+  padding-right: 16px;
+  max-width: 1200px;
+  margin: 0 auto;
+  padding-top: 32px;
+  padding-bottom: 32px;
+}
+
+.bestsellers {
+  display: flex;
+  justify-content: space-between;
+  flex-flow: row wrap;
+}
+.bestsellers::after {
+  content: "";
+  flex: auto;
+}
+
+.bestsellers-item {
+  border: 2px solid #eee;
+  padding: 8px;
+  margin-right: 15px;
+  margin-bottom: 15px;
+  text-align: center;
+  width: 265px;
+  height: 400px;
+  flex-shrink: 0;
+  flex-grow: 0;
+  transition: border-color 400ms ease;
+}
+
+.bestsellers-item:hover {
+  border-color: #222;
+}
+
+.bestsellers-item figure {
+  width: 60%;
+  height: 150px;
+  padding-left: 60px;
+  padding-top: 40px;
+  margin: 0 0 16px;
+  font-size: 0;
+  cursor: pointer;
+}
+
+.item-name {
+  font-size: 14px;
+  font-weight: bold;
+  color: #4c884c;
+  cursor: pointer;
+}
+
+.item-name:hover {
+  color: #333;
+}
+
+.item-price {
+  font-size: 18px;
+  font-weight: bolder;
+  color: #777;
+}
+
+.star-widget {
+  width: 175px;
+  height: 31px;
+}
+
+.star-widget input {
+  display: none;
+}
+
+.star-widget label {
+  font-size: 18px;
+  color: #d3d3d3;
+  padding: 0;
+  float: right;
+  transition: all 0.2s ease;
+}
+
+/* .star-widget > input:checked ~ label, 
+     .star-widget:not(:checked) > label:hover, 
+      .star-widget:not(:checked) > label:hover ~ label {
+  color: #ffd700;
+}
+
+.star-widget > input:checked + label:hover,
+.star-widget > input:checked ~ label:hover,
+.star-widget > label:hover ~ input:checked ~ label,
+.star-widget > input:checked ~ label:hover ~ label {
+  color: #ffed85;
+} */
+
+input:not(:checked) ~ label:hover,
+input:not(:checked) ~ label:hover ~ label{
+  color: #D2691E;
+}
+input:checked ~ label{
+  color: #FF8C00; 
+  /* "checked" not working */
+}
+
+.product-btn {
+  border: 2px solid #ddd;
+  width: 40px;
+  height: 40px;
+  padding: 6px;
+  cursor: pointer;
+  background-color: #fff;
+}
+
+.product-btn:hover {
+  background-color: #222;
+  border: none;
+  transition: background-color 400ms linear;
```

Comment:
transition property should be defined in the class and not in pseudo-class (:hover)